crates/jws/src/lib.rs

```diff
@@ -27,6 +29,10 @@ pub enum JwsError {
     AlgorithmNotFound(String),
     #[error(transparent)]
     CryptoError(#[from] CryptoError),
+    #[error("serde json error {0}")]
```

Comment:
that's right! the 'ole "v2 this thing temporarily" strategy, which I can understand people not being receptive to, if not we can just hold off merging and string a sequence of PRs together, but I figure since it's a tight nit team working and we plan to do this within the next 2 days might as well keep our process time as low as possible

good catch, I removed these changes